src/main/java/net/glowstone/inventory/GlowMetaSkull.java

```diff
@@ -98,10 +98,12 @@ void writeNbt(CompoundTag tag) {
     void readNbt(CompoundTag tag) {
         super.readNbt(tag);
         if (tag.containsKey("SkullOwner")) {
-            if (tag.isString("SkullOwner")) {
-                owner.set(GlowPlayerProfile.getProfile(tag.getString("SkullOwner")).join());
-            } else if (tag.isCompound("SkullOwner")) {
-                owner.set(GlowPlayerProfile.fromNbt(tag.getCompound("SkullOwner")).join());
+            if (!tag.readString(
```

Comment:
I think flipping the key (string) and the function params for "read*" methods would make their usage more readable.

Reply:
Then they can't be variadic, since varargs have to come last. I'll look into how often I'm actually using them with more than one argument, now that the conversions are done.